### PR TITLE
Feature/auto update - Periodically redownload and reload config.json and image from url 

### PR DESCRIPTION
**Pull request by CallMeKass**
*moved worker data into workers.json to prevent private data being shared
*download image and config over http from url
*added auto_update settings to config.json
*separate thread is created for updates

Allows for easy usage of a centralized image and config file. 
Useful for when people running your bot are busy and can't immediately update the image or config themselves.

### CODE REVIEW

**Comment by CallMeKass** [comment]
Just wanted to get this up for visibility. README still needs updated and I'm too tired to deal with the merge conflicts right now.

**Comment by CallMeKass** [comment]
I already have this branch setup to work with auto updates if you want to see a real world config for this: https://github.com/CallMeKass/reddit-transplace-script-2022/tree/testing

File: main.py [hunk 1]
```diff
@@ -140,6 +183,23 @@ def load_image(self):
         logger.info("Loaded image size: {}", im.size)
         self.image_size = im.size
 
+    # Download file from url over http
+    def download_resouce(self, url, destination):
+        r = requests.get(url, allow_redirects=False)
```
Comment (RomainL972) [inline main.py:188]: Can we change allow_redirects to True ? Someone could want to shorten the URL for example which would require a redirection.

Comment (CallMeKass) [reply main.py]: Just updated to include this

**Pull request closed without merging**